src/components/AngelCard.js

```diff
@@ -0,0 +1,67 @@
+//   component that takes props that replace the text in my example.
+import React, { useState, useEffect } from "react"
+import { makeStyles } from "@material-ui/core/styles"
+import Card from "@material-ui/core/Card"
+import CardActions from "@material-ui/core/CardActions"
+import CardContent from "@material-ui/core/CardContent"
+import Button from "@material-ui/core/Button"
+import Typography from "@material-ui/core/Typography"
+
+const useStyles = makeStyles({
+  card: {
+    minWidth: 275,
+  },
+  bullet: {
+    display: "inline-block",
+    margin: "0 2px",
+    transform: "scale(0.8)",
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+  list: {
+    paddingLeft: "1.5em",
+  },
+})
+
+export default function AngelCard(props) {
```

Comment:
I like this style `const AngelCard = (props) => {` then at the bottom `export default AngelCard`

Reply:
👍